docs/pages/versions/unversioned/sdk/permissions.md

```diff
@@ -36,11 +62,20 @@ The following table shows you which permissions correspond to which packages.
 | `SYSTEM_BRIGHTNESS`         | `expo-brightness`                         |
 | `MOTION`                    | `expo-sensors`                            |
 
-## Usage
+### Excluding Android permissions in bare workflow
```

Comment:
Was missing, found the workaround with Eric, and a lot of issues in the React Native repository. Having this documented something really useful I think.